### ISSUE TITLE
hoopR::load_nba_player_box not loading games after 4/2/23

### ISSUE THREAD

**Issue by iksnizez** [sportsdataverse/hoopR#111]
running hoopR::load_nba_player_box(2023) does not return games after 4/2/2023

**Comment by Coop197823** [comment]
I can confirm I am experiencing this issue as well.

**Comment by saiemgilani** [comment]
```r
remotes::install_github("sportsdataverse/hoopR")
```

And try again

**Comment by Coop197823** [comment]
That did the trick, thank you!

**Comment by gabeneslin** [comment]
Hello, I am experiencing a similar issue, when I run hoopR::load_nba_player_box(2023) it does not return any data since the end of the regular season (4/9/ 2023). I have tried 

remotes::install_github("sportsdataverse/hoopR")

as recommended above and I can still not access play in or playoff data. I checked and it looks like this function returns playoff data for previous years, ie hoopR::load_nba_player_box(2022). Has playoff data not been loaded into the database yet?
